azure/package.json

```diff
@@ -111,5 +104,12 @@
 			"lts": "minor",
 			"release/azure/*": "patch"
 		}
+	},
+	"pnpm": {
```

Comment:
Is this temp change?

Reply:
No, this is permanent. Peer dependencies on fluid-framework are never fulfilled since that's an in-repo dependency; we expect customers using some packages to install it as a dependency when using azure-client, but we don't install it alongside azure-client here in the repo.